src/GridLines.tsx

```diff
@@ -222,3 +221,91 @@ const WeekStripes = ({ monthStart, timeScale }: WeekStripesProps) => {
 
   return <g>{lines}</g>
 }
+
+/* ·················································································································· */
+/*  Hour
+/* ·················································································································· */
+
+interface HourLineProps {
+  xPosition: number
+  height?: string
+}
+
+const HourLine = ({ xPosition, height }: HourLineProps) => {
+  const xAxisTheme = useTimelineTheme().xAxis
+  const classes = useMonthViewStyles(xAxisTheme)
+  return (
+    <line
+      className={classes.line}
+      x1={xPosition}
+      y1={0}
+      x2={xPosition}
+      y2={height ? height : '80%'}
```

Comment:
I decided to set it at 80% because if it's 100% then it can stretch over the text. Suggestions on a default welcome.